app/controllers/v0/education_benefits_claims_controller.rb

```diff
@@ -0,0 +1,23 @@
+module V0
```

Comment:
What about Education::V0::ClaimsController? 

Also, namespacing everything under education has the side benefit of keeping all related code under one directory so we can have
- app/controllers/education
- app/models/education
- etc

It's still under discussion whether common/shared endpoints will live at the toplevel v0 or get moved under common/v0 for consistency. I do think we still want to keep the URLs for common endpoints at the top-level (like api.vets.gov/v0/sessions/new)